scripts/bundle-generation/PLATFORM-AWARE-CHANGES.py

```
@@ -0,0 +1,434 @@
+
+    # Update Chart.yaml version if specified
+    if chartVersion:
+        chart_yaml_path = os.path.join(destination_chart_path, "Chart.yaml")
+        try:
+            with open(chart_yaml_path, 'r') as f:
+                chart_yaml = yaml.safe_load(f)
+            chart_yaml['version'] = chartVersion
+            with open(chart_yaml_path, 'w') as f:
+                yaml.dump(chart_yaml, f, width=float("inf"))
+            logging.info(f"Updated Chart.yaml version to: {chartVersion}")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)